lib/src/widgets/slider_picker.dart

```diff
@@ -61,11 +63,11 @@ class _SliderPickerState extends State<SliderPicker> {
                 // Child
                 DecoratedBox(
                     decoration: BoxDecoration(
-                      borderRadius: radius,
-                      border: Border.all(color: Colors.grey),
+                      borderRadius: widget.borderRadius,
+                      border: widget.border ?? Border.all(color: Colors.grey),
```

Comment:
The default border should be treated as the default value, not a fallback value.